test/infamy/netconf.py

```diff
@@ -397,8 +397,8 @@ def delete_xpath(self, xpath):
         libyang.xpath_del(new, xpath)
 
         mod = self.ly.get_module(module)
-        oldd = mod.parse_data_dict(old, no_state=True)
-        newd = mod.parse_data_dict(new, no_state=True)
+        oldd = mod.parse_data_dict(old, no_state=True, validate=False)
+        newd = mod.parse_data_dict(new, no_state=True, validate=False)
```

Comment:
Heh, I think I have the same fix in my branch! :rofl: I have an additional check though before this, but we can take that on my PR later on.

Nice catch!